images/kubectl-build-deploy-dind/scripts/exec-generate-sbom-configmap.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# SBOM config
+TMP_DIR="${TMP_DIR:-/tmp}"
+SBOM_OUTPUT="cyclonedx-json"
+SBOM_OUTPUT_FILE="${TMP_DIR}/${IMAGE_NAME}.cyclonedx.json"
+
+set -x
+# Run sbom and dump to file
+echo "Running sbom scan using syft"
+echo "Image being scanned: ${IMAGE_FULL}"
+set +x
+
+DOCKER_HOST=docker-host.lagoon.svc docker run --rm -v /var/run/docker.sock:/var/run/docker.sock imagecache.amazeeio.cloud/anchore/syft packages ${IMAGE_FULL} -o ${SBOM_OUTPUT} > ${SBOM_OUTPUT_FILE}
+
+if [ -f "${SBOM_OUTPUT_FILE}" ]; then
+    echo "Successfully generated SBOM for ${IMAGE_FULL}"
+
+    SBOM_CONFIGMAP=lagoon-sbom-${IMAGE_NAME}
+
+    set -x
+    # If sbom configmap already exists then we need to update, else create new
+    if kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get configmap $SBOM_CONFIGMAP &> /dev/null; then
+        kubectl --insecure-skip-tls-verify \
+            -n ${NAMESPACE} \
+            create configmap $SBOM_CONFIGMAP \
+            --from-file=${SBOM_OUTPUT_FILE} \
+            -o json \
+            --dry-run=client | kubectl replace -f -
+        kubectl --insecure-skip-tls-verify \
+            -n ${NAMESPACE} \
+            label configmap ${SBOM_CONFIGMAP} \
+            lagoon.sh/insights=sbom \
+            lagoon.sh/project=${PROJECT} \
```

Comment:
lagoon.sh/insights-project

Reply:
all labels to be  prefixed with 'insights'

Reply:
Will keep project/service/environment as usual lagoon labels, but have made insightsType to denote sbom/drutiny etc